mongo/src/main/java/com/redhat/lightblue/mongo/crud/BasicDocSaver.java

```diff
@@ -53,7 +53,7 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(BasicDocSaver.class);
 
-    private int batchSize = 64;
+    private final int batchSize = 64;
```

Comment:
It's definitely a configuration candidate, but @bserdar and I just decided to leave it here for now and move it out in another PR.  Is there a place in LB today to wire up configuration properties?